tests/common/config.sh

```diff
@@ -2,14 +2,16 @@
 
 set -e
 
-build_path=$(pwd)
-while [ "${build_path}" != "/" -a ! -e "${build_path}/.git" ]; do
-	build_path=$(dirname ${build_path})
-done
```

Comment:
From the commit message:

> We don't actually ship the tests with 'make dist'.  So we will
> be exporting tests/ as a separate tarball.  In particular, I want
> to then import this in the debian package.  However, there it will
> be under shadow.git/debian/tests, not shadow.git/tests.

How about having them in the same tarball with the rest of the package, and adding a symlink in Debian with `ln -sT debian/tests tests`?

Reply:
I also think we should export them in the same tarball. Everything makes part of the same solution and having them in a separate file diminishes its importance.

Reply:
I'm perfectly happy with that :)

https://github.com/shadow-maint/shadow/pull/1006 for that part.